Specs/Scene/parseFeatureMetadataLegacySpec.js

```diff
@@ -16,17 +16,17 @@ describe(
         building: {
           properties: {
             name: {
-              type: "STRING",
+              componentType: "STRING",
```

Comment:
I don't think this change is correct, this file is for parsing `EXT_feature_metadata`, not `EXT_mesh_features`. it's one of very few places that should use `type`